Detections/Box/BoxItemSharedToExternalUser.yaml

```diff
@@ -0,0 +1,27 @@
+id: 3b803560-f8a6-4db4-89cb-617d89724ba1
+name: Box - Item shared to external entity
+description: |
+  'Detects when an item was shared to external entity.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: BoxDataConnector
+    dataTypes:
+      - BoxEvents_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+query: |
+  BoxEvents
+  | where EventType == 'COLLABORATION_INVITE'
+  | extend corp_domain = extract(@'@(.*)', 1, SrcUserUpn)
+  | extend accessibleby_domain = tolower(extract(@'@(.*)', 1, AccessibleByLogin))
+  | where corp_domain != accessibleby_domain

```

Comment:
recommend !~ unless you want exact char match.

Reply:
updated

Reply:
doesn't look like this was updated.

Reply:
rule fixed - added toLower() function